package.json

```diff
@@ -94,15 +93,16 @@
     "react-test-renderer": "^16.7.0",
     "recast": "^0.20.4",
     "resolve-url-loader": "^2.2.0",
-    "sass-loader": "^6.0.7",
+    "sass": "^1.37.5",
+    "sass-loader": "^12.1.0",
```

Comment:
Updated to the latest version so I could use `sass` as the underlying sass compiler instead of `node-sass`. I combed through the release notes and didn't find any breaking changes that would affect us as far as I could tell.